src/deque.jl

```diff
@@ -1,7 +1,5 @@
-export push, pop, shift, unshift, insert, deleteat, setindex
-
 @generated function push(vec::StaticVector, x)
-    newtype = similar_type(vec, (length(vec) + 1 ,))
+    newtype = similar_type(vec, Size(length(vec) + 1 ,))
```

Comment:
Minor tidy up: trailing comma no longer required here and in the rest of `deque.jl`.